devops/makeReactBucket/makeReactBucket.bash

```diff
@@ -1,13 +1,41 @@
 #!/bin/bash
```

Comment:
Changes to this file shouldn't be committed, since your action is only using the version found in .github/scripts

Reply:
Thamks for the catch Dan, I found  another issue to might just make another PR here shorty